docs/reference/buildx_build.md

```diff
@@ -828,8 +828,11 @@ $ docker buildx build --platform=darwin .
 --progress=VALUE
 ```
 
-Set type of progress output (`auto`, `plain`, `tty`, `rawjson`). Use `plain` to show container
-output (default `auto`).
+Set type of progress output (`auto`, `quiet`, `plain`, `tty`, `rawjson`). Use
+`plain` to show container output (default `auto`).
+
+> [!NOTE]
+> `--progress=quiet` is the same as `--quiet`.
```

Comment:
nit: what about this formatting instead:

```suggestion
Set type of progress output. Supported values are:

- `auto` (default): uses the `tty` mode if the client is a TTY, or `plain` otherwise
- `tty`: an interactive stream of the output with color and redrawing
- `plain`: prints the raw build progress in a plaintext format
- `quiet`: Suppress the build output and print image ID on success (same as `--quiet`)
- `rawjson`: prints the raw build progress as JSON lines
```

Reply:
:rocket: